src/notifications/generate.ts

```diff
@@ -131,7 +131,8 @@ export const notificationTitleMap: Record<
     return `New post from <b>${userName}</b>, check it out now!`;
   },
   user_given_top_reader: (ctx: NotificationUserTopReaderContext) => {
-    const keyword = ctx.keyword.flags?.title || ctx.keyword.value;
+    const keyword =
+      (ctx.keyword.flags as KeywordFlags)?.title || ctx.keyword.value;
```

Comment:
only issue now is that we need to do this on some other places were we use `Reference` because its `ChangeObject` under the hood. But let's leave it for now.

Reply:
Yep. Although I think these are the only places, which is why build failed and I had to update them.

Reply:
Some fruits of strict type checking 👌